build-system/tasks/e2e/index.js

```diff
@@ -80,20 +80,28 @@ function createMocha_() {
   let reporter;
   if (argv.testnames || argv.watch) {
     reporter = '';
-  } else if (argv.report) {
+  } else if (argv.report || isCircleciBuild()) {
```

Comment:
Will this work for CI builds where `--report` is not being passed in?

Reply:
That is my intention, but I can see why that may be unintuitive. An alternative would be to set the variable in CircleCI (either in the config.yml file for each task or on the dashboard). Do you have a preference?

Reply:
On looking at the code, it looks like `argv.report` only generates the report file and doesn't affect the run in any other way, so I'm okay with leaving this as-is.